metrics/grafana/ticdc.json

```diff
@@ -14,7 +14,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "6.1.6"
+      "version": "7.5.7"
```

Comment:
Whether Grafana in all release branches have been upgraded?

Reply:
yes, all branches affected.